### PR TITLE
Fix missing articles and grammer in docs

### LINKED ISSUE #395: Fix all basic grammar mistakes in the documentation

### PR DESCRIPTION
**Pull request by Jolanrensen**
Same as https://github.com/Kotlin/dataframe/pull/394 but moved inside the repo, so we can work together in the same branch.

Fixed #395 

Edit: Okay, this is too large to fix in 1 simple PR, especially before the 0.11.0 release.
I'll merge the checked changes now and postpone the checks for later. The entire documentation needs a good check, in grammar, clarity and spelling, but this will take longer.

### CODE REVIEW

File: docs/StardustDocs/topics/cast.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 
 **Parameters:**
 * `verify: Boolean = false` —
-  when `true`, throws exception if [`DataFrame`](DataFrame.md) doesn't match the given schema. 
-Otherwise, just change a format type without actual data check.
+  when `true`, the function throws an exception if the [`DataFrame`](DataFrame.md) instance doesn't match the given schema. 
+Otherwise, it just changes the format type without actual data checks.
 
-Use this operation to change a formal type of [`DataFrame`](DataFrame.md)
+Use this operation to change the formal type of a [`DataFrame`](DataFrame.md) instance
```
Comment (zaleslaw) [inline docs/StardustDocs/topics/cast.md:15]: So, better to go through all the docs and add instance to all places where it should be applied

**Comment by zaleslaw** [comment]
@Jolanrensen feel free to fix "instances" issue and merge it ASAP in the master branch without submitting on review, because I need to continue my work on the docs today and tomorrow

File: docs/StardustDocs/topics/apiLevels.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ look pretty similar in the data wrangling DSL.
 
 ## List of Access APIs
 
-Here's a list of all APIs in order to increase safety.
```
[the remark this answers is not in the document]
Comment (Jolanrensen) [reply docs/StardustDocs/topics/apiLevels.md]: No, it really isn't. We say the list is ordered by increasing safety. So the lower on the list, the safer it gets. We describe the order, not the reason ("in order to...").

File: docs/StardustDocs/topics/cast.md (hunk 1 above)
Comment (Jolanrensen) [reply docs/StardustDocs/topics/cast.md:15]: let's do that afterward though. Currently, I'm just going through your changes and checking and replacing grammar there. When I'm done, I'll do a ctrl+r on the entire thing :) Either here or a separate PR

**Comment by Jolanrensen** [comment]
I'll revert the grammar fixes in the code parts (well, running korro will do that). We should probably fix those in a separate PR if we want to. That said, missing articles in quick code comments are less of an issue, since compactness is often key there

**Pull request merged**